music_album.rb

```diff
@@ -24,7 +24,7 @@ def add_music_album
     end
 
     music_albums.push({ 'id' => @id, 'on_spotify' => @on_spotify, 'publish_date' => @publish_date,
-                        'archived' => @archived })
+                        'author' => @author, 'Music_genre' => @genre, 'archived' => @archived })
```

Comment:
Since you added `author` and `genre` to the file, kindly add `label` too to the hash 😊

Reply:
Thanks for the review, Will work on it